plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -303,6 +303,9 @@ class WC_Install {
 		'10.2.0' => array(
 			'wc_update_1020_add_old_refunded_order_items_to_product_lookup_table',
 		),
+		'10.3.0' => array(
```

Comment:
@jorgeatorres, what is the plan when it comes to DB updates and deploying `-beta` releases?  Do we need the versions to be set to the current `-dev`?

Reply:
It's fine to use `10.3.0`. The way it works now, it is required (and will be validated at build time) that any db update added between versions after `-beta.1` receives a new `db_version` that is higher than before, but it can just be `10.3.0-1`, `10.3.0-2`, and so on. If there aren't any other db updates or all added before the feature freeze, `10.3.0` will work just fine.